sw/device/tests/sim_dv/sysrst_ctrl_ec_rst_l_test.c

```diff
@@ -140,12 +142,12 @@ static void check_combo_reset(void) {
 // indicate that the testbench is ready to proceed with the
 // next phase of the test.
```

Comment:
small nit: since we are already changing `wait_next_test_phase`, we might as well update it as the function description as /* */ comments, instead of inline comment.